paddlepaddle/paddlepaddle-native/build.gradle

```diff
@@ -273,33 +273,6 @@ publishing.repositories {
     }
 }
 
-
```

Comment:
We will need this eventually, right?

Reply:
We already have `prepareNativeLib` which essentially doing the same thing